proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -503,8 +503,12 @@
       "data. Defaults to none.")
   protected String traceListenerPorts;
 
+  @Parameter(names = {"--traceJaegerListenerProtocol"}, description = "Protocol used to listen " +
```

Comment:
nit: IMO we can name this something shorter like "traceJaegerProtocol"

Reply:
agreed, let's split into `traceJaegerListenerPorts` (tchannel) and `traceJaegerHttpListenerPorts`

Reply:
Do we want to support a different application via HTTP too? In which case we would add traceJaegerHttpApplicationName as well.  Or even multiple applications, in which case we could add a property like `traceJaegerConfig=14267:tchannel:App1,14268:http:App2`

Reply:
If i understood your question correctly, Jaeger spans sent via any method(http/tchannel) should have "Application" as one of the span tags to distinguish between applications. 
So It is just two ports traceJaegerListenerPorts (tchannel) and traceJaegerHttpListenerPorts (http) as vasily mentioned.

Reply:
yes, and I believe that tag value is pulled from proxy-level Jaeger application name in the config (`traceJaegerApplicationName`), so I'm wondering if we need to expand the config to allow for multiple application names (per protocol or per port)?

Reply:
If customer has multiple applications best would be to ask them to use "Application" tag as a Jaeger process level tag rather than depending on the Proxy level tags.
Don't think we need to handle this case at the proxy level.

Reply:
ah i see. thanks for clarifying

Reply:
added `traceJaegerHttpListenerPorts` and removed `traceJaegerProtocol`